bin/pycbc_make_skymap

```diff
@@ -321,11 +321,27 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, f_upper, sample_rate,
     else:
         doc.save(ligolw_file_path)
 
+    # Defining the approximant to feed into BAYESTAR
+
+    row = WaveformArray.from_kwargs(
+	    mass1=opt.mass1,
+	    mass2=opt.mass2,
+	    spin1z=opt.spin1z,
+	    spin2z=opt.spin2z)
+
+    approximant = waveform.bank.parse_approximant_arg(opt.approximant, row)[0]
+
+    # BAYESTAR uses TaylorF2 instead of SPAtmplt
+
+    if approximant == 'SPAtmplt':
+	approximant = 'TaylorF2'
+
```

Comment:
Just a minor request for indentation: line 337 should be indented with 4 spaces relative to the previous one.

Reply:
Personal taste, I would also remove the empty line 335, as the comment refers specifically to a very short block of code.